locales/en-US/app.ftl

```diff
@@ -1,23 +1,20 @@
 siteName = Firefox Test Pilot
 
-// Page titles, put in the <title> HTML tag.
-[[pageTitle]]
+## Page titles, put in the <title> HTML tag.
 pageTitleDefault = Firefox Test Pilot
 pageTitleLandingPage = Firefox Test Pilot
 pageTitleExperimentListPage = Firefox Test Pilot - Experiments
 pageTitleExperiment = Firefox Test Pilot - {$title}
```

Comment:
Should we add a Term for Firefox Test Pilot?

Reply:
It might make sense, but in case it should be in a different issue/discussion, given how many translations we would lose.